### PR TITLE
merging CNN work

### PR DESCRIPTION
**Pull request by cleebp**

### CODE REVIEW

**Comment by Rishabh2693** [comment]
have made some edit, Could you please try running this neural net on your
system. Also can you play around with value of number of nodes in neural
network.
Further we have used initially in our code a learning rate of 0.001, which
requires atleast 1000 epochs to optimize to a stable output, I have added a
learning rate of 0.01 with 100 epochs. and also changed some values of
nodes in the net.
